packages/core/test/test_utils.js

```diff
@@ -43,3 +53,9 @@ export function describeRepeated(title, fn) {
     );
   }
 }
+
+export async function submitForm(node) {
```

Comment:
Why does this function (and the function wrapped in `act`) need to be async?

Reply:
apparently it didn't :) still haven't quite figured out when and when not you need `act` and when to make it async.